docs/en-US/Remove-PsDscResource.md

```
@@ -0,0 +1,171 @@
+
+## INPUTS
+
+## OUTPUTS
+
+### System.Management.Automation.PSObject
+
+{{ Fill in the Description }}
+
+## NOTES
+
+For more details, go to module repository at: https://github.com/Gijsreyn/PSDSC.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)